helm/mds/tests/auth-metrics-jwt-audience_test.yaml

```diff
@@ -0,0 +1,64 @@
+suite: test authetication jwt audience metrics
```

Comment:
nit but you might wanna rename to "authentication"

Reply:
appreciate the 👀 @janedotx :) thx!